src/sensors/utils.rs

```diff
@@ -22,9 +29,15 @@ impl ProcessTracker {
     /// let tracker = ProcessTracker::new(5);
     /// ```
     pub fn new(max_records_per_process: u16) -> ProcessTracker {
+        let regex_cgroup_docker = Regex::new(r"^/docker/.*$").unwrap();
+        let regex_cgroup_kubernetes = Regex::new(r"^/kubepods.slice/.*$").unwrap();
```

Comment:
On the cluster I was testing with the cgroups file has a different format. Could the regex be more generic to support both formats?

```
#/proc/193876/cgroup
1:name=systemd:/kubepods/burstable/pod7a8cbc91-66e9-4303-88df-513f77240233/acd77757d49868ead1f706f901271e737594d0e11cec86d4bfa4de45a0512938
```

The cluster was kubernetes v1.20.2 installed using kubeadm on ubuntu 20.10 with docker 20.10.2